app/controllers/general.php

```diff
@@ -713,7 +733,7 @@ function router(App $utopia, Database $dbForConsole, callable $getProjectDB, Swo
         }
 
         switch ($class) {
-            case 'Utopia\Exception':
+            case 'Utopia\Servers\Exception':
```

Comment:
hmm was the namespace changed?

Reply:
Yes, but it should be `Utopia\Http\Exception`